### ISSUE TITLE
[A11y] Missing type of homescreen's "More options" button 

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/focus-android#2943]
## Steps to reproduce
1. Activate Talkback.
2. On the homescreen, no page loaded, tap the Menu button.

### Expected behavior
Talkback should say the type of the "More options" button.

### Actual behavior
Homescreen Menu button doesn't say if it's a menu/button, just "More options".
In a tab with a loaded page, it says "More options button" correctly.

### Device information

*Android device:
LG Nexus 5 (Android 6.0.1)
Google Pixel (Android P beta)

*Focus version: 6.3
